admin/js/boldgrid-backup-admin-settings-autoupdate.js

```diff
@@ -183,19 +223,30 @@ BOLDGRID.SETTINGS = BOLDGRID.SETTINGS || {};
 		 */
 		_toggleHelp: function( e ) {
 			var id = $( this ).attr( 'data-id' );
-
+			var target = $( '.table-help[data-id="' + id + '"]' );
```

Comment:
Don't have 2 `var` statements in a row. Combine them into 1. https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#declaring-variables-with-var

Reply:
resolved